assets/stylesheets/units/_buttons.scss

```diff
@@ -0,0 +1,43 @@
+.button {
+  @include bold-font(20);
+
+  -webkit-appearance: none;
+  background-color: shade($green, 12.5%);
+  border: 0;
+  color: $white;
+  cursor: pointer;
+  display: inline-block;
+  letter-spacing: .5px;
+  margin: 0 ($baseline-grid-unit * 6) ($baseline-grid-unit * 3) 0;
```

Comment:
Fixed ;)

It's not that we're afraid of margins it's more a case of element flow. Using that selector allows elements to sit at the top of parent elements and allow that element to handle the spacing. There's a good [article](http://alistapart.com/article/axiomatic-css-and-lobotomized-owls) by Heydon Pickering which explains the benefit of it. 

If we use bottom margin on all then you get what we have now where there is double the spacing because the container element is also setting the same spacing.